microwave/README.md

```diff
@@ -21,10 +21,11 @@ It features a virtual piano UI enabling you to play polyphonic microtonal melodi
 cargo install -f microwave
 ```
 
-You might need to install additional dependencies required by Nannou. For me, the following setup worked:
+You might need to install additional dependencies required by Nannou. For me - working on Ubuntu 18.04 LTS - the following setup worked:
 
 ```bash
-sudo apt install pkg-config libx11-dev
+sudo apt install pkg-config libx11-dev libasound2-dev
+sudo apt-get install libvulkan1 mesa-vulkan-drivers vulkan-utils
```

Comment:
Thanks for your PR. :-) For consistency, I would either write `apt-get` or `apt` everywhere

Reply:
Thanks! I'll change this to `apt`.